packages/types/surveys/types.ts

```diff
@@ -2487,3 +2488,14 @@ export const ZSurveyCopyFormValidation = z.object({
 });
 
 export type TSurveyCopyFormData = z.infer<typeof ZSurveyCopyFormValidation>;
+
+export const ZSurveyMigrateFormValidation = z.object({
+  products: z.array(
+    z.object({
+      product: z.string(),
+      environments: z.array(z.string()),
```

Comment:
`environments` in this schema is not being used anywhere, can we maybe remove this? Or is it required?